censys/config.py

```diff
@@ -0,0 +1,68 @@
+"""
+Interact with the config file.
+"""
+
+import os
+from pathlib import Path
+from configparser import ConfigParser, NoOptionError
+
+from censys import __version__
+
+DEFAULT = "DEFAULT"
+
+censys_path = os.path.join(str(Path.home()), ".censys")
```

Comment:
While this isn't _exactly_ the XDG spec, `.config` is where this stuff should go.
```suggestion
censys_path = os.path.join(str(Path.home()), ".config/censys")
```

Reply:
Added in 7a0073f